minesweeper.py

```diff
@@ -266,17 +269,10 @@ def flagging(event):
     "The force is strong with this one.",
 ]
 
-
-def highscore(mins, secs):
-    converted_to_seconds = mins * 60 + secs
-    for time in top_10_times:
-        if converted_to_seconds < time and len(top_10_times) < 10:
-            top_10_times.insert(
-                index, converted_to_seconds
-            )  # HOW DO I GET THE INDEX OF "TIME" IN FOR LOOP?
-        elif converted_to_seconds < time and len(top_10_times >= 10):
-            top_10_times.remove(top_10_times[-1])
-            top_10_times.insert(index, converted_to_seconds)  # SAME AS ABOVE
+def highscore():
+    with open(where_this_file_is / 'high_scores.json', 'r') as high_scores:
+        score_list=json.load(current_game.game_time.strftime("%M:%S"), high_scores)
```

Comment:
- This doesn't work. `json.load` takes only one argument
- Result variable should be `json_dict`, not `score_list`
- `highscore()` function should be called somewhere, quite early in the program so that the rest of the code uses whatever it loaded from the file
- This needs error handling in case the file doesn't exist